docs/generate-time-series/javascript.rst

```diff
@@ -0,0 +1,241 @@
+.. _gen-ts-javascript:
+
+==========================================
+Generate time series data using Javascript
+==========================================
+
+This tutorial will show you how to generate some :ref:`experimental time series
+data <gen-ts>` by consuming telemetry data from the `International Space Station`_
```

Comment:
maybe "[...] how to generate some experimental time series data from information about the International Space Station[...]"